specification/synapse/data-plane/Microsoft.Synapse/preview/2021-06-01-preview/entityTypes/Dataset.json

```diff
@@ -467,8 +467,8 @@
       ],
       "properties": {
         "level": {
-          "description": "The ZipDeflate compression level.",
-          "$ref": "#/definitions/CompressionLevel"
+          "type": "object",
+          "description": "The ZipDeflate compression level."
```

Comment:
I'm confused by this change, is it a "Expression or String" again? If that's the case, could you update the description to clarify it?

Reply:
Yes, updated the description